src/EditorFeatures/Core/GoToDefinition/AbstractGoToDefinitionService.cs

```diff
@@ -41,10 +41,10 @@ internal abstract class AbstractGoToDefinitionService : IGoToDefinitionService
 
         public bool TryGoToDefinition(Document document, int position, CancellationToken cancellationToken)
         {
-            // First try to compute the referenced symbol and attempt to go to definition for the symbol.
+            // Try to compute the referenced symbol and attempt to go to definition for the symbol.
             var symbolService = document.GetLanguageService<IGoToDefinitionSymbolService>();
-            var (symbol, span) = symbolService.GetSymbolAndBoundSpanAsync(document, position, cancellationToken).WaitAndGetResult(cancellationToken);
-            if (symbol == null)
+            var (symbol, _) = symbolService.GetSymbolAndBoundSpanAsync(document, position, cancellationToken).WaitAndGetResult(cancellationToken);
+            if (symbol is null)
```

Comment:
Is "symbol is null" preferred? Did I miss the memo?

Reply:
Here it doesn't matter because `symbol` is `ISymbol`. I can change back.
`is null` is safer as it is always object equality (no risk of an overridden equality operator).

Reply:
Well, we've been asking for operator overloading on == between ISymbol for years... ;-)

Reply:
It's fine, I was seeing that in the compiler codebase and wasn't sure why. In general, for IDE code if there was an operator overload we'd probably want to call it. I know the compiler is persnickety about that but we don't care.

Reply:
I love "is null"  because it means no confusoin for me ever.  I would prefer it for all ref-equal-null checks in the IDE.

Reply:
OK, then ignore my opinion. I weep for our language where this is a thing.